subtree_imports.go

```diff
@@ -5,6 +5,7 @@ package main
 
 import _ "github.com/taskcluster/taskcluster-cli/apis"
 import _ "github.com/taskcluster/taskcluster-cli/config"
+import _ "github.com/taskcluster/taskcluster-cli/config-cmd" // commands for config
```

Comment:
It's starting to make sense. I'm gonna keep the current package as it is for now, but we'll talk about this again before merging `install-cobra` into `master`.